packages/expo-file-system/src/next/index.ts

```diff
@@ -1 +1 @@
-export { Directory, File, write } from './FileSystem';
+export { Directory, File, write, download } from './FileSystem';
```

Comment:
Please, no 😅 We will end up with so many functions like this, it makes more sense to scope them

Reply:
Is it OK if I do `export *` here then? I don't think we'll have more than a few of these module functions.